### PR TITLE
fix(InfiniteGrid): add defense code on scroll

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#263 

## Details
If the infiniteGrid haven't element and 'scroll' event is fired, we face a problem.

## Preferred reviewers
@kishu @netil

### CODE REVIEW

File: src/infiniteGrid.js [hunk 1]
```diff
@@ -317,6 +317,9 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 			if (prevScrollTop < scrollTop) {
 				if ($.isEmptyObject(this._bottomElement)) {
 					this._bottomElement = this.getBottomElement();
+					if (this._bottomElement == null) {
```
Comment (netil) [inline src/infiniteGrid.js]: if `_buttomElement` should be checked strictly as `null` use === instead ==.
undefined can be treated true in below comparison.
```js
undefined == null;
```

Comment (sculove) [inline src/infiniteGrid.js]: @netil I want to check both `undefined` and `null`. thanks

Comment (netil) [inline src/infiniteGrid.js]: LGTM

**Comment by kishu** [comment]
LGTM

**Pull request merged**